modules/keycode/keycode.js

```diff
@@ -1,5 +1,6 @@
 /**
  * Enumerates the supported keys.
+ * NOTE: The maps represents actual keys on the keyboard not chars. 
```

Comment:
Is this key set related somehow to what robotjs supports ? Maybe some link to robotjs docs ?